src/common/components/repository-row/index.js

```diff
@@ -350,7 +370,11 @@ export class RepositoryRowView extends Component {
       );
     }
 
-    return <TickIcon />;
+    return (
+      <span onClick={this.onConfiguredClick.bind(this)}>
```

Comment:
This doesn't change a cursor to pointer which makes it really hard to discover that it can be clicked.